### PR TITLE
fix: add <samsorrahman>

### LINKED ISSUE #1718: Samsor Rahman
### Profile screenshot

[Photo](https://photos.google.com/photo/AF1QipMFdsayypEj053tLvv6oXIJL8Nd-_M_ZcoMPiBH)

### PR DESCRIPTION
**Pull request by samsorrahman**
#1718

resolves #1718 

## Checklist for a profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by PiseAtharva — COMMENTED** [review]
Hey Samsor, it seems you have altered some details related to Bylickilabs. The issueId is valid as [1573](https://github.com/EddieHubCommunity/awesome-github-profiles/issues/1573) like you have mentioned. But its best to not alter the information unless Bylickilabs or maintainers approves you for that.

**Review by PiseAtharva — COMMENTED** [review]
As for your personal profile json, everything's good to go, just the image does not seems to load on Internet (maybe update the url for that ?). If you can afford try to use github's cloud service (_user-images.githubusercontent_) for hosting your image ( let me know if you want help with that )

**Comment by samsorrahman** [comment]
[Click Me](https://www.dropbox.com/s/7qhkcseefcnli5q/my%20gito.PNG?dl=0)

you can use the above URL to solve the issue Thanks

**Comment by samsorrahman** [comment]
[This is also the url for image](https://www.dropbox.com/s/7qhkcseefcnli5q/my%20gito.PNG?dl=0)

**Comment by Panquesito7** [comment]
There's no need for a dropbox link. You can just use a snipping tool, copy it into a GitHub message, copy the image's link, and use it in the image URL field. 🙂
If you need any help with doing that, let us know.

**Comment by samsorrahman** [comment]
how to do this would you like to instruct @Panquesito7

**Comment by Panquesito7** [comment]
> how to do this would you like to instruct @Panquesito7

If you're on Windows, you can search for the Snipping Tool application and take a screenshot of your profile.
If you're using another operating system, I'm sorry, but I can't help much, as I use Windows. 😅

**Comment by samsorrahman** [comment]
git pro

**Comment by samsorrahman** [comment]
Now you can merge it

**Comment by samsorrahman** [comment]
@PiseAtharva  is it right now or not?
please confirm me 
thank you so much

File: profiles/Bylickilabs.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "image": "https://user-images.githubusercontent.com/109308073/209451382-c2c90f77-0c2a-45dc-bd06-1f5418910dd9.jpg",
-  "issueId": 1588,
+  "image": "https://user-images.githubusercontent.com/109308073/208296882-ea239594-81ea-419f-ae6f-e3470ccc9a3e.png",
```
Comment (Panquesito7) [inline profiles/Bylickilabs.json:2]: Please revert the changes in this file. Let us know if you need any help. 🙂

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/Bylickilabs.json (hunk 1 above)
Comment (samsorrahman) [reply profiles/Bylickilabs.json:2]: Done @Panquesito7 
Thank for helping

**Review by samsorrahman — COMMENTED** [review]
I have revert the changes

File: profiles/Bylickilabs.json (hunk 1 above)
Comment (Panquesito7) [reply profiles/Bylickilabs.json:2]: Hmm, the changes are still there. 🤔

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

The changes are still reflected in the `profiles/Bylickilabs.json`. Please revert them
